packages/venia-ui/lib/components/WishlistPage/wishlist.js

```diff
@@ -62,6 +83,19 @@ const Wishlist = props => {
                         {contentToggleIcon}
                     </button>
                 </div>
+                <WishlistListActionsDialog
```

Comment:
Team just had [a conversation about this](https://magento.slack.com/archives/G8N21AQTV/p1615564627083800), and it sounds like Informed DynamicFields could be a solution here.

Reply:
I have looked into DynamicFields to be able to turn this into a single Dialog, but it looks as though that option will not allow some flexibility needed to customize the Dialog header title, header actions, and mask actions between the two Dialogs currently in place. Considering this, and parity with `wishlistItem`, I will leave this pattern as is for now, but can re-visit if team decides otherwise.